catroid/src/main/java/org/catrobat/catroid/utils/ToastUtil.java

```diff
@@ -39,18 +39,10 @@ public static void showError(Context context, String message) {
 		createToast(context, message, true);
 	}
 
-	public static void showError(Context context, int messageId) {
-		createToast(context, context.getResources().getString(messageId), true);
-	}
-
 	public static void showSuccess(Context context, String message) {
 		createToast(context, message, false);
 	}
 
-	public static void showSuccess(Context context, int messageId) {
-		createToast(context, context.getResources().getString(messageId), false);
-	}
-
```

Comment:
Ahm, I'm not sure about these changes. Why did you remove these methods?

Reply:
just unused Methods.

Reply:
Yes :smile: My question _Why did you remove these methods?_ was meant to be much more general. So I also implied changes like
```diff
-ToastUtil.showError(…, R.string.foo);
+ToastUtil.showError(…, getString(R.string.foo))
```
in my statement. imo this helper method is definitely useful because you already pass in a context object (from which you get your resources).

___
Btw, please take a look at the hint about RTL at https://jira.catrob.at/browse/CAT-2043?focusedCommentId=18602&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18602